volumes.c

```diff
@@ -163,7 +161,7 @@ int pv_volume_mount(struct pv_volume *v)
 	}
 
 	if (v->disk && !v->disk->def && !v->disk->mounted) {
-		ret = pv_disks_mount_handler(v->disk, "mount");
+		ret = pv_disk_mount_handler(v->disk, "mount");
```

Comment:
This is the same point where we should mount disks. Maybe changing the name of the function if handler is no longer used, then decide whether to use handler or not inside.